[2021-09-30 19:26:00,841] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: etl_dag.kafka_producer 2021-09-28T00:01:00+00:00 [queued]>
[2021-09-30 19:26:00,865] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: etl_dag.kafka_producer 2021-09-28T00:01:00+00:00 [queued]>
[2021-09-30 19:26:00,866] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2021-09-30 19:26:00,866] {taskinstance.py:1088} INFO - Starting attempt 2 of 1
[2021-09-30 19:26:00,866] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2021-09-30 19:26:00,878] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): kafka_producer> on 2021-09-28T00:01:00+00:00
[2021-09-30 19:26:00,881] {standard_task_runner.py:52} INFO - Started process 1710412 to run task
[2021-09-30 19:26:00,885] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'etl_dag', 'kafka_producer', '2021-09-28T00:01:00+00:00', '--job-id', '434', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/etl_dag.py', '--cfg-path', '/tmp/tmpwe4pqiyk', '--error-file', '/tmp/tmpzpab0a6j']
[2021-09-30 19:26:00,886] {standard_task_runner.py:77} INFO - Job 434: Subtask kafka_producer
[2021-09-30 19:26:00,942] {logging_mixin.py:104} INFO - Running <TaskInstance: etl_dag.kafka_producer 2021-09-28T00:01:00+00:00 [running]> on host kseniya-B85M-D3V
[2021-09-30 19:26:01,027] {taskinstance.py:1300} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=etl_dag
AIRFLOW_CTX_TASK_ID=kafka_producer
AIRFLOW_CTX_EXECUTION_DATE=2021-09-28T00:01:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-09-28T00:01:00+00:00
[2021-09-30 19:26:01,028] {pass_op.py:3} INFO - pass_op
[2021-09-30 19:26:01,028] {python.py:151} INFO - Done. Returned value was: None
[2021-09-30 19:26:01,069] {taskinstance.py:1204} INFO - Marking task as SUCCESS. dag_id=etl_dag, task_id=kafka_producer, execution_date=20210928T000100, start_date=20210930T162600, end_date=20210930T162601
[2021-09-30 19:26:01,111] {taskinstance.py:1265} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-09-30 19:26:01,135] {local_task_job.py:149} INFO - Task exited with return code 0
